.github/renovate.json5

```diff
@@ -4,7 +4,8 @@
     "config:recommended",
     "schedule:weekly",
     "group:allNonMajor",
-    ":disablePeerDependencies",
+    ":disablePeerDependencies", 
+    "regexManagers:biomeVersions"
```

Comment:
Since we pin `@biomejs/biome`, do we want renovate to auto update the biome schema version? it might not work between versions.

Reply:
There's a chance a person updates biome and forgets to update the schema version.

In case that happens, renovate will update it when it raises the usual PR. That's my understanding from the person that added the feature

Reply:
Ah so the version in `biome.json` will always match the version that's installed in `node_modules`? That sounds great then 👍